packages/anima-cli/src/helpers/storybook.ts

```diff
@@ -0,0 +1,41 @@
+import fs from 'fs';
+import path from 'path';
+import { API_URL } from '../constants/api';
+
+const JS_EXTENSIONS = ["js", "jsx", "ts", "tsx", "vue"];
+
+export const getJSFiles = (folder: string) => {
+    const JSFiles: string[] = [];
+
+    for (const item of fs.readdirSync(folder)) {
+        const fullPath = path.join(folder, item);
+        if (fs.lstatSync(fullPath).isDirectory()) {
+            JSFiles.push(...getJSFiles(fullPath));
+        } else {
+            const extension = item.split(".").pop()
+            if (extension && JS_EXTENSIONS.includes(extension) && item.charAt(0) === item.charAt(0).toUpperCase()) {
+                JSFiles.push(fullPath);
+            }
+        }
+    }
+
+    return JSFiles;
+};
+
+export const generateStorybookConfig = async (file: string, token: string) => {
+    const res = await fetch(`${API_URL}/generate_storybook/`, {
```

Comment:
for non-rest endpoint that are remote procedure calls we use the convention of `/rpc/` and without `/` suffix

```suggestion
    const res = await fetch(`${API_URL}/rpc/generate_storybook`, {
```